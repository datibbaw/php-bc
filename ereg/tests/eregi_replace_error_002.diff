004+ string(71) "helloshelloohellomhelloehello helloshellothellorhelloihellonhelloghello"
005- Warning: eregi_replace(): REG_EMPTY in %s on line %d
006+ Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
007+ 
008+ Warning: preg_replace(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
010- Warning: eregi_replace(): REG_EPAREN in %s on line %d
013+ Warning: preg_replace(): Compilation failed: missing terminating ] for character class at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
015- Warning: eregi_replace(): REG_EBRACK in %s on line %d
018+ Warning: preg_replace(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
020- Warning: eregi_replace(): REG_EPAREN in %s on line %d
023+ Warning: preg_replace(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
025- Warning: eregi_replace(): REG_BADRPT in %s on line %d
028+ Warning: preg_replace(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
030- Warning: eregi_replace(): REG_BADRPT in %s on line %d
033+ Warning: preg_replace(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
035- Warning: eregi_replace(): REG_BADRPT in %s on line %d
038+ Warning: preg_replace(): Compilation failed: nothing to repeat at offset 1 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
040- Warning: eregi_replace(): REG_BADRPT in %s on line %d
041- bool(false)
042+ string(11) "some string"
044+ Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
045+ string(71) "helloshelloohellomhelloehello helloshellothellorhelloihellonhelloghello"
046+ 
047+ Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
048+ string(71) "helloshelloohellomhelloehello helloshellothellorhelloihellonhelloghello"
045- Warning: eregi_replace(): REG_BADBR in %s on line %d
050- Warning: eregi_replace(): REG_EMPTY in %s on line %d
052+ Warning: preg_replace(): Compilation failed: numbers out of order in {} quantifier at offset 5 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
055- Warning: eregi_replace(): REG_EMPTY in %s on line %d
056+ string(11) "somd string"
057- 
058+ Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
059+ 
060+ Warning: preg_replace(): No ending delimiter '/' found in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
065+ Warning: preg_replace(): Compilation failed: range out of order in character class at offset 4 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
067+ Done
058- Deprecated: Function eregi_replace() is deprecated in %s on line %d
059- 
060- Warning: eregi_replace(): REG_BADBR in %s on line %d
061- bool(false)
062- 
063- Deprecated: Function eregi_replace() is deprecated in %s on line %d
064- 
065- Warning: eregi_replace(): REG_ERANGE in %s on line %d
066- bool(false)
067- 
068- Deprecated: Function eregi_replace() is deprecated in %s on line %d
069- 
070- Warning: eregi_replace(): REG_EESCAPE in %s on line %d
071- bool(false)
072- 
073- Deprecated: Function eregi_replace() is deprecated in %s on line %d
074- 
075- Warning: eregi_replace(): REG_ERANGE in %s on line %d
076- bool(false)
077- Done