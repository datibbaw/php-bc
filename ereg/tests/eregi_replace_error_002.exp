*** Testing eregi_replace() : bad REs ***

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_EBRACK in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_ERANGE in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_EESCAPE in %s on line %d
bool(false)

Deprecated: Function eregi_replace() is deprecated in %s on line %d

Warning: eregi_replace(): REG_ERANGE in %s on line %d
bool(false)
Done