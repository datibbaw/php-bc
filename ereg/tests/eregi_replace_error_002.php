<?php
/* Prototype  : proto string eregi_replace(string pattern, string replacement, string string)
 * Description: Replace regular expression 
 * Source code: ext/standard/reg.c
 * Alias to functions: 
 */

echo "*** Testing eregi_replace() : bad REs ***\n";
var_dump(eregi_replace("", "hello", "some string"));
var_dump(eregi_replace("c(d", "hello", "some string"));
var_dump(eregi_replace("a[b", "hello", "some string"));
var_dump(eregi_replace("c(d", "hello", "some string"));;
var_dump(eregi_replace("*", "hello", "some string"));
var_dump(eregi_replace("+", "hello", "some string"));
var_dump(eregi_replace("?", "hello", "some string"));
var_dump(eregi_replace("(+?*)", "hello", "some string"));
var_dump(eregi_replace("h{256}", "hello", "some string"));
var_dump(eregi_replace("h|", "hello", "some string"));
var_dump(eregi_replace("h{0}", "hello", "some string"));
var_dump(eregi_replace("h{2,1}", "hello", "some string"));
var_dump(eregi_replace('[a-c-e]', 'd', "some string"));
var_dump(eregi_replace('\\', 'x', "some string"));
var_dump(eregi_replace('([9-0])', '1', "some string"));
echo "Done";
?>
