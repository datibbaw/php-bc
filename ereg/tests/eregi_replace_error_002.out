*** Testing eregi_replace() : bad REs ***

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
string(71) "helloshelloohellomhelloehello helloshellothellorhelloihellonhelloghello"

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: missing terminating ] for character class at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: nothing to repeat at offset 1 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
string(11) "some string"

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
string(71) "helloshelloohellomhelloehello helloshellothellorhelloihellonhelloghello"

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
string(71) "helloshelloohellomhelloehello helloshellothellorhelloihellonhelloghello"

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: numbers out of order in {} quantifier at offset 5 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157
string(11) "somd string"

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): No ending delimiter '/' found in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)

Deprecated: Function eregi_replace() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 157

Warning: preg_replace(): Compilation failed: range out of order in character class at offset 4 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 162
bool(false)
Done